client/src/pages/Feed.js

```diff
@@ -293,7 +330,17 @@ const Feed = () => {
       .catch(error => {
         postsDispatch({ type: ERROR_POSTS });
       })
-  }, []);
+  }, [ props ]);
+
+  useEffect(() => {
+    if (posts.status === SET_POSTS && posts.posts.length) {
+      const likePost = sessionStorage.getItem("likePost");
+
+      if (likePost) {
```

Comment:
Can you clarify why we are calling `handlePostLike ` here?

Reply:
If a guest user clicks like, it goes to the login page and when logged user comebacks to feed page, like is saved.

Reply:
Ah ok. But why not handle this logic in `PostSocial`? I think it makes more sense to handle this behavior in that component than in the Feed. What do you think?

Reply:
Done.